docs/helm-chart/quick-start.rst

```diff
@@ -217,6 +217,13 @@ Example below adds ``lxml`` PyPI package.
           --set images.airflow.repository=my-image \
           --set images.airflow.tag=0.0.1
 
+If you put your image in a private docker repository, the name of secret can be set at registry.secretName
```

Comment:
If you follow this quick-start and use `kind load`, you don't have to do provision a secret, right?